packages/katello/katello/katello-backup-rotate-tar.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+## For this script it's advisable to use a shell, such as Bash,
+## that supports a TAR_FD value greater than 9.
+
+# This script taken from the tar info doc
+
+name=`expr ${TAR_ARCHIVE} : '\(.*\)\..*'`
+volnum=`printf "%04d" ${TAR_VOLUME}`
+case ${TAR_SUBCOMMAND} in
+-c)       ;;
+-d|-x|-t) test -r ${name:-$TAR_ARCHIVE}.part${volnum:-$TAR_VOLUME} || exit 1
+          ;;
+*)        exit 1
+esac
+
+echo ${name:-$TAR_ARCHIVE}.part${volnum:-$TAR_VOLUME} >&${TAR_FD}
+echo " Switched to multi-volume ${name:-$TAR_ARCHIVE}.part${volnum:-$TAR_VOLUME}"
```

Comment:
I'm not a bash expert, but I would think you would want `${name:-$TAR_ARCHIVE}.part${volnum:-$TAR_VOLUME}` as a variable since I see it used multiple places.

Reply:
done